fjord/feedback/views.py

```diff
@@ -470,8 +498,7 @@ def feedback_router(request, product=None, version=None, channel=None,
     elif product in PRODUCT_OVERRIDE:
         # If the product is really a form name, we use that
         # form specifically.
-        view = PRODUCT_OVERRIDE[product]
-        product = None
+        view = PRODUCT_OVERRIDE[product]['view']
```

Comment:
I think you're not using this right. The PRODUCT_OVERRIDE is really an (ab)use of the product field in the url to direct to a specific form. In that case, the product variable holds a form name--not a product.

I think instead of doing this, you should follow the firefoxos path in the previous `if` block where it checks and does something different that's product-specific.